core/src/services/gcs/config.rs

```diff
@@ -53,6 +53,8 @@ pub struct GcsConfig {
     pub disable_vm_metadata: bool,
     /// Disable loading configuration from the environment.
     pub disable_config_load: bool,
+    /// Enable versioning for the bucket.
+    pub enable_versioning: bool,
```

Comment:
Let's just remove this config. I will change the bucket settings to have a versioning-enabled one.